### PR TITLE
Signaling Markets Contracts

### PR DESCRIPTION
**Pull request by emaG3m**

### CODE REVIEW

File: contracts/DecisionLib.sol [hunk 1]
```diff
@@ -25,6 +25,7 @@ library DecisionLib {
   function closeDecisionMarkets(Decision storage self) returns (uint winningMarketIndex) {
     uint previousBalance = self.token.balanceOf(this);
     self.futarchyOracle.close();
+    winningMarketIndex = 3;
```
Comment (mikec) [inline contracts/DecisionLib.sol]: this probably needs to be removed?

Comment (emaG3m) [reply contracts/DecisionLib.sol]: ohhhh

File: contracts/FutarchySignalingOracleFactory.sol [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    /// @param upperBound Lower bound for event outcome
+    /// @param marketMaker Market maker contract
+    /// @param fee Market fee
+    /// @param tradingPeriod Trading period before decision can be determined
+    /// @param startDate Start date for price logging
+    /// @return Oracle contract
+    function createFutarchyOracle(
+        ERC20Gnosis collateralToken,
+        Oracle oracle,
+        uint8 outcomeCount,
```
[the remark this answers is not in the document]
Comment (emaG3m) [reply contracts/FutarchySignalingOracleFactory.sol]: if we remove this param, we change the interface, hence `futarchyOracleFactory` will have a different interface than `futarchySignalingOracleFactory`. probably not worth it!!

File: contracts/kit/FutarchyKit.sol [hunk 3]
```diff
@@ -36,4 +36,4 @@ contract FutarchyKit is KitBase, APMNamehash, IsContract {
 
         return dao;
     }
-}
\ No newline at end of file
+}
```
[the remark this answers is not in the document]
Comment (emaG3m) [reply contracts/kit/FutarchyKit.sol]: will do it in another one

File: contracts/FutarchySignalingOracleFactory.sol (hunk 2 above)
[the remark this answers is not in the document]
Comment (mikec) [reply contracts/FutarchySignalingOracleFactory.sol]: 👍 good point!

**Comment by mikec** [comment]
looking good! 👍 👍

**Pull request merged**